atomate/vasp/firetasks/run_calc.py

```diff
@@ -147,6 +148,11 @@ def run_task(self, fw_spec):
         elif job_type == "full_opt_run":
             jobs = VaspJob.full_opt_run(vasp_cmd, auto_npar=auto_npar, ediffg=self.get("ediffg"),
                                         max_steps=5, half_kpts_first_relax=False)
+        elif job_type == "neb":
```

Comment:
@computron Re principle of do no harm, custodian has always defaulted half kpts to False. I don't remember it ever being True by default. There may be other parameters that were set that way, but half_kpts was not one of them.

In any case, many such arguments go out of the window with something like Custodian and atomate, which basically changes parameters to fix things and prescribes a "good" approach to doing a set of calculations respectively. Our NEB workflow is a prescription for an *efficient* and *accurate* NEB calculation. In our opinion, most other people are doing it badly or inefficiently. They can override the defaults we set, but it is what the doctor recommended.